src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    
```

Comment:
let's remove this empty line

Reply:
Actually, I have tried it three or fore times, as you can see without success.

Reply:
It`s kind of bag I suppose.